### PR TITLE
Simplify PhpFilesFinder

### PR DESCRIPTION
**Pull request by staabm**
I couldn't measure any perf difference.

it might result in a perf boost when https://github.com/symfony/symfony/pull/50087 is contained in symfony

### CODE REVIEW

File: src/ValueObject/StaticNonPhpFileSuffixes.php [hunk 1]
```diff
@@ -11,13 +11,4 @@ final class StaticNonPhpFileSuffixes
      */
     public const SUFFIXES = ['neon', 'yaml', 'xml', 'yml', 'twig', 'latte', 'blade.php', 'tpl'];
```
Comment (TomasVotruba) [inline src/ValueObject/StaticNonPhpFileSuffixes.php]: I wonder how is this class needed now. What prevents it from being removed?

Comment (staabm) [reply src/ValueObject/StaticNonPhpFileSuffixes.php]: it seems you are fine with dropping this constant and you don't mind BC.

therefore I moved the constant to the last remainig consumer and removed the whole class

File: src/NonPhpFile/NonPhpFileProcessor.php [hunk 2]
```diff
@@ -18,6 +18,12 @@
 
 final class NonPhpFileProcessor implements FileProcessorInterface
 {
+    /**
+     * @var string[]
+     */
+    private const SUFFIXES = ['neon', 'yaml', 'xml', 'yml', 'twig', 'latte', 'blade.php', 'tpl'];
```
Comment (staabm) [inline src/NonPhpFile/NonPhpFileProcessor.php:24]: > '#Relative file path "blade\.php" is not allowed, use absolute one with __DIR__#'

the error message does not make sense for a constant array,... ?

Comment (staabm) [reply src/NonPhpFile/NonPhpFileProcessor.php:24]: https://github.com/symplify/phpstan-rules/pull/35

Comment (TomasVotruba) [reply src/NonPhpFile/NonPhpFileProcessor.php:24]: It does. This one should be ignored in phpstan like the previous case.

Comment (staabm) [reply src/NonPhpFile/NonPhpFileProcessor.php:24]: ignored

**Comment by TomasVotruba** [comment]
Thanks :+1:

**Pull request merged**